From 691dbd26f1e0aa64daa72e4d85ea8310ceded789 Mon Sep 17 00:00:00 2001
From: Nikhil V <quic_nprakash@quicinc.com>
Date: Fri, 27 Sep 2024 15:33:43 +0530
Subject: [PATCH 1/1] QCLINUX: Revert "UPSTREAM: arm64: dts: qcom: sa8775p:
 enable the inline crypto engine"

This reverts commit 9bae7529cb7dd3a2b846ef5ed43ba5415244a435.

Signed-off-by: Nikhil V <quic_nprakash@quicinc.com>
Upstream-Status: Inappropriate [Revert]
---
 arch/arm64/boot/dts/qcom/sa8775p.dtsi | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/sa8775p.dtsi b/arch/arm64/boot/dts/qcom/sa8775p.dtsi
index 48f59e45c8e0..3e4af03e5cb8 100644
--- a/arch/arm64/boot/dts/qcom/sa8775p.dtsi
+++ b/arch/arm64/boot/dts/qcom/sa8775p.dtsi
@@ -1684,7 +1684,6 @@
 					<0 0>,
 					<0 0>,
 					<0 0>;
-			qcom,ice = <&ice>;
 			status = "disabled";
 		};
 
@@ -1706,13 +1705,6 @@
 			status = "disabled";
 		};
 
-		ice: crypto@1d88000 {
-			compatible = "qcom,sa8775p-inline-crypto-engine",
-			             "qcom,inline-crypto-engine";
-			reg = <0x0 0x01d88000 0x0 0x8000>;
-			clocks = <&gcc GCC_UFS_PHY_ICE_CORE_CLK>;
-		};
-
 		stm: stm@4002000 {
 			compatible = "arm,coresight-stm", "arm,primecell";
 			reg = <0x0 0x4002000 0x0 0x1000>,
-- 
2.17.1

